src/pybind/mgr/dashboard/controllers/smb.py

```diff
@@ -0,0 +1,129 @@
+
+# -*- coding: utf-8 -*-
+
+import logging
+import json
+
+from typing import Dict, List, Union
+
+
+from dashboard.controllers._permissions import CreatePermission, DeletePermission
+from smb import results
+from smb.enums import AuthMode, SMBClustering, Intent
+
+from .. import mgr
+from ..security import Scope
+from . import APIDoc, APIRouter, ReadPermission, RESTController
+
+logger = logging.getLogger('controllers.smb')
+
+
+@APIRouter('/smb/cluster', Scope.SMB)
+@APIDoc("SMB Cluster Management API", "SMB")
+class SMBCluster(RESTController):
+    _resource: str = 'ceph.smb.cluster'
+
+    @ReadPermission
+    def list(self) -> List[Dict]:
+        """
+        List smb clusters
+        """
+        res = mgr.remote('smb', 'show', [self._resource])
+        return res['resources'] if 'resources' in res else res
```

Comment:
while testing the UI PR, I saw that if there is only one response, it just returns the json which breaks the table. @phlogistonjohn just checking shall we just have a single type (list of resources) everytime, even if it has only one item?

```suggestion
        return res['resources'] if 'resources' in res else [res]
```

Reply:
Yep, behaviour is slightly different for one item vs many. That's the why of this return `res['resources'] if ...`

Reply:
Ah right, yes. To be consistent on the return type, it should be [res]. 
Updated it and added another test to cover this scenario